kernel/api/src/main/java/org/sakaiproject/tasks/api/Task.java

```diff
@@ -61,7 +61,7 @@
     @Column(name = "DESCRIPTION", length = 255, nullable = false)
     private String description;
 
-    @Column(name = "\"SYSTEM\"", nullable = false)
+    @Column(name = "TASK_SYSTEM", nullable = false)
```

Comment:
I think I prefer SYSTEM_TASK, tbh.

Reply:
Man, SYSTEM from the TASK table is a TASK's SYSTEM right? TASK_SYSTEM

😃

Reply:
No, it's a task generated by the system. So, assignments creates a task, that's a system task.

Reply:
Really, it should have been an ENUM.

Reply:
So weird man, renamed to SYSTEM_TASK.